pkg/cmd/attestation/attestation.go

```diff
@@ -13,9 +14,13 @@ import (
 func NewCmdAttestation(f *cmdutil.Factory) *cobra.Command {
 	root := &cobra.Command{
 		Use:     "attestation [subcommand]",
-		Short:   "Work with artifact attestations",
+		Short:   "(BETA) Work with artifact attestations",
```

Comment:
For lack of a better word, this `(BETA)` doesn't feel right. I like that we're trying to lower expectations given that it's highly likely we might have to iterate on the interface, but maybe let's drop it from the `Short`

```suggestion
		Short:   "Work with artifact attestations",
```

and tweak the Long heredoc.